tests/regression/.env

```diff
@@ -73,6 +80,11 @@ ALS_KAFKA_MAX_REQUEST_SIZE_=10485760
 # Symlink pointing to current ULS database
 ULS_CURRENT_DB_SYMLINK=FS_LATEST.sqlite3
 
+ULS_ALARM_SMTP_INFO=
+ULS_ALARM_EMAIL_TO=wcc-afc-swdev.pdl@broadcom.com
+ULS_ALARM_ALARM_INTERVAL_HR=0
+ULS_ALARM_SUCCESS_MAX_AGE_HR=9
+ULS_ALARM_BEACON_INTERVAL_HR=0
```

Comment:
Default single-service stuff better be put to its Dockerfile.
Or at least detailed comments to settings better be added